### PR TITLE
fixing string placement in opensearch_dashboards_overview

### PR DESCRIPTION
**Pull request by mukatay1**
### Description
The goal of this task is to enhance the readability and maintainability of the Overview component in the OpenSearch Dashboards project. By completing this task, the Overview component will be more maintainable, easier to understand, and adhere to the project's coding standards. This will facilitate future development and reduce the likelihood of introducing bugs or regressions.
 
### Issues Resolved
Improved readability and maintainability of the Overview component
[#3049](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3049)
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by ashwin-pc** [comment]
@mukatay1 can you fix the lint errors in the code and add a changelog entry for the change?

**Comment by mukatay1** [comment]
> can you fix the lint errors in the code and add a changelog entry for the change?

Hi, yes I can.

**Comment by mukatay1** [comment]
> @mukatay1 can you fix the lint errors in the code and add a changelog entry for the change?

@ashwin-pc @abbyhu2000  Can you please review this pr?

File: CHANGELOG.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multi DataSource] UX enhancement on Data source management stack ([#2521](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2521))
 - [Multi DataSource] UX enhancement on Update stored password modal for Data source management stack ([#2532](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2532))
 - [Monaco editor] Add json worker support ([#3424](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3424))
+- Fixing string placement in opensearch_dashboards_overview ([#3670](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3670))
```
Comment (joshuarrrr) [inline CHANGELOG.md:209]: minor style nits - we prefer the imperative "use" or "fix" over "using" or "fixing". I also updated to be more reflective of the new feature.
```suggestion
- [Overview] Use branding.applicationTitle as page header ([#3670](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3670))
```

Comment (mukatay1) [reply CHANGELOG.md:209]: Thank you for your feedback and updates, I'll make sure to use the imperative form and keep the new feature in mind for future responses.

File: src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx [hunk 2]
```diff
@@ -149,10 +149,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features, bran
...
-            defaultMessage="OpenSearch Dashboards"
-            id="opensearchDashboardsOverview.header.title"
-          />
+          branding.applicationTitle || (
+            <FormattedMessage
+              defaultMessage="OpenSearch Dashboards"
+              id="opensearchDashboardsOverview.header.title"
+            />
+          )
         }
```
Comment (abbyhu2000) [inline src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx:158]: I believe we can just utilize the `<FormattedMessage>` like below. The default of branding.applicationTitle is "OpenSearch Dashboards".

```
title={
            <FormattedMessage
              defaultMessage={branding.applicationTitle}
              id="opensearchDashboardsOverview.header.title"
            />
     }
```

**Review by abbyhu2000 — COMMENTED** [review]
I believe we can just utilize the `<DefaultMessage>` like below. The default is branding.applicationTitle is already`OpenSearch Dashboards`

```
title={
          <FormattedMessage
            defaultMessage={branding.applicationTitle}
            id="opensearchDashboardsOverview.header.title"
          />
        }
```

File: src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx [hunk 3]
```diff
@@ -149,10 +149,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features, bran
...
-          <FormattedMessage
-            defaultMessage="OpenSearch Dashboards"
-            id="opensearchDashboardsOverview.header.title"
-          />
+          branding.applicationTitle || (
+            <FormattedMessage
+              defaultMessage="OpenSearch Dashboards"
+              id="opensearchDashboardsOverview.header.title"
+            />
+          )
```
Comment (abbyhu2000) [inline src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx:157]: What do you think of the above suggestion? It is aligned with previous application title configuration: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/6b42669f1168644806a9d4231077746405623c6a/src/plugins/home/public/application/components/welcome.tsx#L237

Comment (mukatay1) [reply src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx:157]: Its sounds great

**Comment by abbyhu2000** [comment]
@mukatay1 Do you think you are able to make changes to the PR based on the suggestions and also rebase this PR?

**Comment by joshuarrrr** [comment]
@mukatay1 I'm converting this PR to a draft for now - when you have a chance to work on it again, you can take it out of draft mode.

**Pull request closed without merging**